### ISSUE TITLE
Method WordFontAttributes does not work

### ISSUE THREAD

**Issue by czchao** [sirfz/tesserocr#68]
Hi, I want to use method `WordFontAttributes` to get font information of recognized words, but I failed. The image is
<img width="259" alt="test" src="https://user-images.githubusercontent.com/4580068/29117131-a88bc110-7d2f-11e7-94bb-4d6fc9448883.png">
and my code is 
```
with PyTessBaseAPI() as api:
    api.SetImageFile('test.png')
    api.Recognize()
    iterator = api.GetIterator()
    print(iterator.WordFontAttributes())
```
It outputs `None`. Is there something wrong in my code?

**Comment by sirfz** [comment]
Have you tried different images? It worked on the image I tested on (try [eurtext.tif](https://github.com/sirfz/tesserocr/blob/master/tests/eurotext.tif)).

**Comment by czchao** [comment]
I tried my code on the `eurotext.tif`, but it still output `None`. Is the output result related with the environment? 
The OS is `Ubuntu 16.04`, python version is `3.6.1`, tesseract version is
```
tesseract 4.00.00alpha
 leptonica-1.74.4
  libjpeg 8d (libjpeg-turbo 1.4.2) : libpng 1.6.31 : libtiff 4.0.6 : zlib 1.2.8
```
, and tesserocr version is `2.2.2`

**Comment by sirfz** [comment]
Did some further testing and found weird behavior. If I use the 4.00 tessdata, there are no font attributes returned for the image. However, if I use the 3.04.00 tessdata, font attributes are returned. You can bring this issue up with the tesseract team if you want to look into it.

In the mean time, I recommend you use the latest stable version (3.05.01) with the 3.04.00 tessdata.

**Comment by sirfz** [comment]
Closing this issue given it's tesseract-related.

**Comment by NehaPandey58** [comment]
Method WordFontAttributes is not working for me also. I am using 
tesseract 4.0.0-beta.1-270-g5a56
 leptonica-1.74.4
  libjpeg 6b : libpng 1.6.34 : zlib 1.2.11
Kindly help.

**Comment by kvdesai** [comment]
I landed up here while trying to figure out how to determine FontSize using tesserocr.

The tesseract issue #1074 referenced above seems to have been fixed. 
https://github.com/tesseract-ocr/tesseract/pull/1173/commits

However, with tesserocr the method WordFontAttributes is still returning None. 
Following is my setup 

```
Ubuntu 16.04, x64

$ tesseract --version
tesseract 4.1.0-rc1-263-gb107
leptonica-1.78.0
libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.4.2) : libpng 1.2.54 : libtiff 4.0.6 : zlib 1.2.8 : libwebp 0.4.4 : libopenjp2 2.3.0
Found SSE

tesserocr.__version__
2.4.0

```
Any suggestions to fix this?

**Comment by Jamxscape** [comment]
> I landed up here while trying to figure out how to determine FontSize using tesserocr.
> 
> The tesseract issue #1074 referenced above seems to have been fixed.
> https://github.com/tesseract-ocr/tesseract/pull/1173/commits
> 
> However, with tesserocr the method WordFontAttributes is still returning None.
> Following is my setup
> 
> ```
> Ubuntu 16.04, x64
> Python 3.7.1
> 
> $ tesseract --version
> tesseract 4.1.0-rc1-263-gb107
> leptonica-1.78.0
> libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 1.4.2) : libpng 1.2.54 : libtiff 4.0.6 : zlib 1.2.8 : libwebp 0.4.4 : libopenjp2 2.3.0
> Found SSE
> 
> tesserocr.__version__
> 2.4.0
> ```
> 
> Any suggestions to fix this, or a work around to determine Font Size and Is Bold?

Have you ever solved this problem?I Use tesseract 4.1.1 and tessdata 4.0 It doesnot work also,and just return to None
